storage/driver.go

```diff
@@ -73,4 +73,8 @@ type Driver interface {
 	// ListSnapshots returns an array of snapshot names provided a maximum number
 	// of snapshots to be returned. Any error will be returned.
 	ListSnapshots(DriverOptions) ([]string, error)
+
+	// ShowMapped shows any volumes that belong to volplugin on the host, in
```

Comment:
@erikh 

A quick clarification. Are you planning to use this API for showing the mount information in one of the volcli outputs? I ask as `Mount` struct seems very host specific, do you think for multihost environment this API shall dump some host-info as well (like a host label or hostname)? Or will that be taken care at the volmaster level?